docs/source/kinds_of_types.rst

```diff
@@ -356,6 +356,43 @@ check against ``None`` in the if condition.
 
     ``--strict-optional`` is experimental and still has known issues.
 
+.. _noreturn:
+
+The NoReturn type
+*****************
+
+Mypy also provides an experimental support for functions that supposed to
```

Comment:
that _are_ supposed

Also, I think you should remove "an" in "an experimental support".